Cookbook/Style-guide/README.md

```diff
@@ -782,6 +780,22 @@ class TimeMachine {
 }
 ```
 
+Use `private`, `fileprivate`, `internal`, and `public` access-level modifiers on each function implemented in an extension, rather than having a single access-level modifier for the entire extension.
```

Comment:
I'm nitpicking here, but the paragraph says "on each function" and the exemple shows a `var` not a `func` 😉and better be clear anyway 

```suggestion
Use `private`, `fileprivate`, `internal`, and `public` access-level modifiers on each property, type or function declared in an `extension`, rather than having a single access-level modifier for the entire `extension`.
```